ReactLatestApp/src/App.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
+import "./App.css";
+import decode from "jwt-decode";
+import LoginPage from "./components/login";
+import Home from "./components/home";
+import Error from "./components/error";
+
+// const checkAuth = () => {
```

Comment:
@Vinay-Sunny you can remove all thee comments as well.. take up these changes as next PR.